ENG-1946 - feat: introduce React package documentation (#143)
Basis-Theory/docs: source/includes/elements/react/_basis_theory_react.md

# BasisTheoryReact

> `BasisTheoryReact` adds React-specific capabilities on top of `BasisTheory` class. All [API features](/api-reference) and [Elements Services](#elements-services) are still available, plus the methods below.

Reviewer: At least add `the` in front of `BasisTheory class` here, but again, it would be nice to have a better word to describe this other than class.

> `BasisTheoryReact` adds React-specific capabilities on top of `BasisTheory` class. All [API features](/api-reference) and [Elements Services](#elements-services) are still available, plus the methods below.

Author: ```suggestion
`BasisTheoryReact` adds React-specific capabilities on top of the `BasisTheory` class. All [API features](/api-reference) and [Elements Services](#elements-services) are still available, plus the methods below.
```

## getElement

```jsx
import { useBasisTheory } from '@basis-theory/basis-theory-react';
import { TextElement } from '@basis-theory/basis-theory-react/elements';

const ssnMask = [/\d/, /\d/, /\d/, "-", /\d/, /\d/, "-", /\d/, /\d/, /\d/, /\d/];

const MyForm = () => {
  const { bt } = useBasisTheory('test_1234567890', { elements: true });

  const submit = async () => {
    const fullName = bt.getElement('fullName');
    const ssn = bt.getElement('ssn');
    const tokens = await bt.tokenize({
      fullName,

>       ssn,

Reviewer: It's a little more verbose, but maybe this should create the ssn token with `social_security_number` type, and possibly override the privacy settings for fullName to classify it as `pii`. Maybe that's too much for this particular guide, but we should talk about the ability to do that somewhere..

Author: I think that is a great idea for this
https://linear.app/basis-theory/issue/ENG-1986/create-collect-pii-with-react-guide

    });
  }

  return <BasisTheoryProvider bt={bt}>
    <TextElement id="fullName" placeholder="Full name" aria-label="Full name" />
    <TextElement id="ssn" placeholder="SSN" aria-label="Social Security Number" mask={ssnMask} transform={/[-]/} />
    <div>
      <button type="submit" onClick={submit} disabled={!bt}>Submit</button>
    </div>
  </BasisTheoryProvider>;
}
```

> Gets the underlying Element instance to be used for tokenization. Returns `undefined` if can't match an Element with the provided `id`.

Reviewer 2: ```suggestion
Gets the underlying Element instance to be used for tokenization. Returns `undefined` if an Element with the provided `id` does not exist.
```

Parameter | Required | Type      | Description
--------- | -------- | --------- | ------------

> `id`      | true     | *string*  | The Element's required `id` property

Reviewer 3: ```suggestion
`id`      | true     | *string*  | The `id` of the Element instance
```
